routers/utils/utils.go

```diff
@@ -15,3 +18,57 @@ func RemoveUsernameParameterSuffix(name string) string {
 	}
 	return name
 }
+
+// ParseRepoOrderByType returns the OrderByType represented by the given string.
+func ParseRepoOrderByType(s string) models.RepoOrderByType {
```

Comment:
This just likes a construct function so it should be with the type.

Reply:
I decided to put `ParseRepoOrderByType` and `ToQueryString` in `module/util/`, because they deal with how the orderings are represented in query strings. IMO, `models/` should not deal with things like query strings; instead, `models/` only should contain backend code (interacting with the DB, defining models).